odl/tomo/geometry/parallel.py

```diff
@@ -0,0 +1,412 @@
+# Copyright 2014, 2015 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Parallel beam geometries."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from abc import ABCMeta
+from math import cos, sin  # , acos, atan2, sqrt
+from future import standard_library
+from future.utils import with_metaclass
```

Comment:
This has some issues (it crashes some versions of pylint), instead use the one in `odl.util`.